gradle.properties

```diff
@@ -23,7 +23,7 @@ gradleToolingApiVersion=8.10.2
 grailsGradlePluginVersion=7.0.0-SNAPSHOT
 groovyVersion=4.0.23
 gspVersion=7.0.0-SNAPSHOT
-hamcrestVersion=3.0
+hamcrestVersion=1.3
```

Comment:
hamcrest was downgraded from 3 to 1.3?

Reply:
3.0 is the wrong version. It was never upgraded to 3.0

Reply:
`org.hamcrest.Factory` is is not in >= 2